### ISSUE TITLE
Using noSubdir option results in "Is a directory" error

### ISSUE THREAD

**Issue by hjerabek** [DoctorEvidence/lmdb-store#15]
Using Node 12.18.3, I tried creating an `lmdb-store` with the `noSubdir` option. Instead of a single lmdb file, a directory gets created and an "Is a directory" error (code 21) is thrown. The following snippet should allow to reproduce the problem:

```javascript
var db=require("lmdb-store").open({
    path:"mydata",
    maxDbs:1,
    noSubdir:true
});
console.log(db.get("foo"));
console.log(db.putSync("foo","bar @ "+Date.now()));
console.log(db.get("foo"));
```

PS: The respective option works as expected when using `node-lmdb` itself.

**Comment by kriszyp** [comment]
Should be fixed in 0.8.3, thanks for reporting.

**Comment by hjerabek** [comment]
Thanks for the quick fix.

**Comment by kriszyp** [comment]
I actually noticed another related issue with noSubdir being unable to create a new file on non-Windows OS that should be fixed with 0.8.4.
